QuangF.NV/lession_1/lib/components/customGradientText.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+class GradientText extends StatelessWidget {
+  const GradientText(this.text);
+
+  final String text;
+
+  static const topColor = Color(0xFFFF6D00);
+  static const bottomColor = Color(0xFFFC2D00);
```

Comment:
tạm thời thì color e cũng cho vào 1 file nhé ( sau vào dự án có bộ gen name thì cũng tương tự),  để về sau tiện refactor hoặc reuse được.  Tên color là tên biến, e lấy tên màu ở đây https://chir.ag/projects/name-that-color/#6195ED